src/components/Calculator.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable react/destructuring-assignment */
```

Comment:
- [ ] You have disabled so many Linters in your codebase, Linters help us to write quality and readable codes, kindly fix those errors instead of disabling the linters